README.md

```diff
@@ -1771,4 +1802,25 @@ UTC is determined using two components:
 
 The Unix Time Stamp is a way of tracking time as a running total of seconds between the Unix Epoch and the present. The count starts at the Unix Epoch on 01/01/1970 at UTC and runs until the present moment at any given time around the world.
 
-Unfortunately, on 19/01/2038 the Unix Time Stamp will cease to work due to a 32-bit overflow. Before this moment any dependent applications will either need to adopt a new convention for time stamps or be migrated to 64-bit systems (which would buy the time stamp a little more time).
\ No newline at end of file
+Unfortunately, on 19/01/2038 the Unix Time Stamp will cease to work due to a 32-bit overflow. Before this moment any dependent applications will either need to adopt a new convention for time stamps or be migrated to 64-bit systems (which would buy the time stamp a little more time).
```

Comment:
assuming you store unix timestamps in 32-bit integers - solution is just to use `long` instead of `int` :)